src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -3233,6 +3302,34 @@ boolean using(Context context, DeclarationList declList) throws ParserException
         return true;
     }
 
+    private String explicitCast(Type from, Type to, boolean override) {
+        String ecmn = explicitCastMethodName(to.javaName);
```

Comment:
If you're talking about these functions, you never call them to produce a downcast, so we can rename them to upcast() and upcastMethodName(), right?

Reply:
Done.